docs/services/compute/virtual-machines.md

```diff
@@ -42,221 +42,337 @@ An Azure virtual machine gives you the flexibility of virtualization without hav
 
-2. **Regular Right-Sizing Reviews:**
+1. **Regular right-sizing reviews:**
    - Conduct regular reviews of VM sizes based on performance assessments and Azure Advisor recommendations. Right-size VMs to align with actual usage patterns.
 
-3. **Scheduled Shutdowns:**
+<!---
+Might also be good to call out what recommendations are available in Advisor and include some ARG queries for pulling them.
+-->
+
+
+2. **Scheduled shutdowns:**
    - Identify and schedule automated VM shutdowns during idle periods. Implement Azure Automation runbooks for efficient management of startup and shutdown routines.
 
-4. **Continuous Monitoring and Alerts:**
+3. **Continuous monitoring and alerts:**
    - Implement continuous monitoring using Azure Monitor and set up
 
+4. - **Automated tagging with Azure policy:** Simplify and standardize tagging by utilizing Azure Policy to automatically apply tags to VMs. This streamlines the allocation process, ensuring adherence to tagging standards.
 
 ### Workload management
 
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
+
+<!---
+ add new feature to a tool: something in the workload management section about moving VMs between regions based on regional pricing. That's probably a good opportunity for a workbook, PBI report, or AOE recommendation :-)
+
+ There's also an opportunity for us to build some tooling to recommend cm-resource-parent tags you should apply to your resources, then offering an approval workflow to apply them on your behalf.
+
+
+-->
 
-1. **Regular Performance Reviews:**
+
+1. **Regular performance reviews:**
    - Conduct regular reviews of VM performance metrics. Adjust VM SKU based on changing workload patterns.
 
-2. **Automation Implementation:**
+2. **Automation implementation:**
+<!---
+- FYI - Autostop is part of the workload management capability. Just calling it out in case I forget by the time I get down to that section :-)
+-->
+
    - Implement automation for routine workload management tasks. This includes auto stop and start VMs based on schedule.
 
-3. **Continuous Monitoring:**
+3. **Continuous monitoring:**
    - Establish a continuous monitoring process using Azure Monitor and Log Analytics. Proactively address performance issues to maintain optimal workload management.
 
 
 ### Licensing + SaaS
 
+<!-- TODO: Add AHB and update based on F2 licensing guidance-->
+
+
+
 ### Commitment discounts
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
+
+<!---
+Let's get more specific and actionable. ARG queries, PBI reports, etc.
+-->
 
-Commitment-based discounts, such as Reserved Instances (RIs) and Azure Savings Plans, offer significant cost savings for organizations utilizing Virtual Machines (VMs) in Azure.
+Commitment-based discounts, such as Reserved Instances (RIs) and Azure Savings Plans, offer significant cost savings for organizations utilizing VMs in Azure.
```

Comment:
nit: Technically, it's "reservation" and not "reserved instance". Using "savings plan" in lowercase for consistency.
```suggestion
Commitment discounts, such as reservations and savings plans, offer significant cost savings for organizations utilizing VMs in Azure.
```